packages/react-components/src/components/MessageThread.tsx

```diff
@@ -244,6 +244,9 @@ export interface MessageThreadStrings {
   /* @conditional-compile-remove(data-loss-prevention) */
   /** String for policy violation message removal details link */
   blockedWarningLinkText: string;
+  /* @conditional-compile-remove(file-sharing) */
```

Comment:
@jimchou-dev can correct me if I am wrong but I think we also want a `/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */` here as well. The teams file sharing use the same rails so it should also be applicable

Reply:
Yes correct and this applies to other places as well to add both
`/* @conditional-compile-remove(file-sharing) @conditional-compile-remove(teams-inline-images-and-file-sharing) */`

Reply:
updated